__tests__/lib/prepublish.js

```diff
@@ -0,0 +1,51 @@
+/* globals afterEach, beforeEach, describe, it, expect */
+
+const mkdirp = require('mkdirp')
+const rimraf = require('rimraf')
+
+const prepublish = require('../../lib/prepublish')
+const util = require('../../lib/util')
+const {buildDir, mockDir} = require('../test-utils/util.js')
+
+describe('prepublish', () => {
+  beforeEach((done) => mkdirp(buildDir, done))
+  afterEach((done) => rimraf(buildDir, done))
+
+  it('should transpile a directory of files', () => {
+    const results = prepublish({
+      entries: util.parseEntries([`${mockDir}:${buildDir}`]),
+      env: 'development'
+    })
+
+    expect(results.length === 1).toBeTruthy()
```

Comment:
Try to fully utilize the `expect` API because in the testing output, this will not show the true difference between expected and observed.  A better way to write this would be:

```javascript
expect(results.length).toBe(1)
```